docs/nodes/bridge-node.mdx

```diff
@@ -43,7 +43,7 @@ From an implementation perspective, Bridge Nodes run two separate processes:
 2. Celestia Node ([see repo](https://github.com/celestiaorg/celestia-node))
 
     * **Celestia Node** augments the above with a separate libp2p network that
-      serves data availability sampling requests. The team sometimes refers to
+      serves data availability sampling requests. The team sometimes refer to
```

Comment:
I think refers is correct

Reply:
yeah, thank you. i'm honestly not quite sure where these all came from

Reply:
8b333632760b273351b2cc7bb9cc77b0186aee97